### ISSUE TITLE
Blocked by Content Security Policy

### ISSUE THREAD

**Issue by hethhhhh** [uBlockOrigin/uMatrix-issues#196]
### Prerequisites

- [X] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
    - [x] I also searched the existing issues at <https://github.com/gorhill/uMatrix/issues>
- [X] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uMatrix](https://old.reddit.com/r/uMatrix/)
- I tried to reproduce the issue when...
    - [X] uMatrix is the only extension
    - [X] uMatrix with default lists/settings
    - [X] using a new, unmodified browser profile
- [X] I am running the latest version of uMatrix
- [X] I checked the [documentation](https://github.com/gorhill/uMatrix/wiki) to understand that the issue I report is not a normal behavior
- [X] I used the logger to rule out that the issue is caused by my ruleset

### Description

https://i.imgur.com/5qQngsG.png

I cannot whitelist whatever is blocking the loading, since the uMatrix filtering is turned off for the domain, it still does something, so far no clue what

The only way to get around "Blocked by Content Security Policy" is to disable uMatrix, then everything loads properly.

### A specific URL where the issue occurs

https://www.icloud.com/

### Steps to Reproduce

1. create a new default profile in Firefox
2. install uMatrix
3. navigate to https://www.icloud.com/

### Ruleset

default ruleset on installation

### Supporting evidence

https://i.imgur.com/5qQngsG.png

### Your environment

* uMatrix version: 1.4.0
* Browser Name and version: Firefox 70.0 / 69.0.2
* Operating System and version: Windows 8.1 / Ubuntu 16.04

**Comment by gorhill** [comment]
If using default setting, you are still spoofing REFERER header, it shows the switch as enabled in your screenshot.

**Comment by uBlock-user** [comment]
Duplicate of https://github.com/uBlockOrigin/uMatrix-issues/issues/130

**Comment by gorhill** [comment]
The report here made me realize there is an issue though, the logger is showing that the `REFERER` header is spoofed, but only the first time the page load. It does not show `REFERER` spoofing on subsequent reload of the page. I need to fix this because the usual advice of looking at the logger may not be helpful in the current case.

**Comment by uBlock-user** [comment]
> but only the first time the page load. It does not show REFERER spoofing on subsequent reload of the page.

That's always been the case, so I thought that was by-design, so didn't report.

**Comment by uBlock-user** [comment]
> . It does not show REFERER spoofing on subsequent reload of the page. 

Badge count also is not shown on subsequent reload of the page alongwith fyi..
